cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/store/TriggerKeys.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.internal.app.runtime.schedule.store;
+
+import co.cask.cdap.proto.id.DatasetId;
+
+/**
+ * Common static methods to handle trigger keys.
+ */
+public class TriggerKeys {
```

Comment:
Also, maybe better to call this `Triggers` since we have `Trigger` class, but no `TriggerKey` class (in general the class name + `s` to denote util for that class).

Another suggestion is we may want to call this `Schedulers` as the place to host simple util method for Scheduler?